### PR TITLE
Fix consonantCount and add skeleton and tests for convertOuncesToLiters

### LINKED ISSUE #671: Fix countConsonants method
There is currently a method called `countConsonants ` that lives in `/lib/<methodName>.js`.
It is incomplete and needs to be fixed!

This methods receives a string as input and returns the number of consonants in the given string.

### Acceptance Criteria:
1. Running `yarn test countConsonants ` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by astone123**
Closes #671

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/consonantCount.js [hunk 1]
```diff
@@ -4,8 +4,23 @@
  * @return {Number}
  */
 
+const isConsonant = (char) =>
+  (char !== "A" && char !== "E" && char !== "I" && char !== "O" && char !== "U" && char !== " ");
```
Comment (ktilcu) [inline lib/consonantCount.js]: This already is it's own method in `./lib`. Could you use that please?

File: test/convertOuncesToLiters.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { convertOuncesToLiters } = require("../lib");
+
+describe("convertOuncesToLiters", () => {
+  test("returns the number of liters from ounces", () => {
+    expect(convertOuncesToLiters(1)).toBe(0.0295735);
+    expect(convertOuncesToLiters(10.5)).toBe(0.3105221);
+    expect(convertOuncesToLiters(32)).toBe(0.946353);
+  });
+});
```
Comment (ktilcu) [inline test/convertOuncesToLiters.test.js]: Could you cover some of the edge cases? What if I pass a negative number or a string or an array?

**Comment by astone123** [comment]
Hey @ktilcu,

I added a commit with your requested changes!

Thanks

**Comment by ktilcu** [comment]
Looks good! @astone123 it would be really helpful if you create a new issue for your new method.
